Seohee/Lv2/수식 최대화.js

```diff
@@ -0,0 +1,37 @@
+function solution(expression) {
+    let result = 0;
+    // 수식 6가지의 경우의 수만 존재
+    const orders = [
+        ['+', '-', '*'],
+        ['+', '*', '-'],
+        ['-', '*', '+'],
+        ['-', '+', '*'],
+        ['*', '-', '+'],
+        ['*', '+', '-'],
+    ];
+    // 숫자들과 연산자들 배열 따로 생성
+    const numbers = expression.split(/[^0-9]/).map((strNumber) => parseInt(strNumber));
+    const operators = expression.split('').filter((data) => "*-+".includes(data));
```

Comment:
```suggestion
    const numbers = expression.split(/\D/g);
    const operators = expression.match(/\D/g);
```
이렇게 표현하면 더 간결하게 표현할 수 있고, 복잡도도 더 낮아질거같아요.